package.json

```diff
@@ -40,7 +39,7 @@
     "jsdom": "3.0.0",
     "jshint": "^2.6.3",
     "react": "^0.14.0",
-    "react-router": "^1.0.3",
+    "react-router": "^2.4.0",
```

Comment:
@remarkablemark thanks for getting this fixed. I've published v3.4.1 with your changes.

@JustinMGaiam can you verify by pulling the latest (npm install)?